### PR TITLE
Implemented input value logging for authentication views

### PR DESCRIPTION
**Pull request by sankettank66**
As per requirements, I have implemented input value capturing in the following authentication views:

- `/forgot-password`
- `/login`
- `/admin-reg`
- `/user-reg`
- `/set-new-password`

Additionally, I have implemented proper validation for the code with robust error handling. Upon successful submission, the input values will be logged in the console.

## Summary by CodeRabbit

- **New Features**
	- Enhanced user experience in authentication components (Forgot Password, Login, Register Admin, Register User, Set New Password) with improved form validation and error handling.
	- Added password validation checks and state management for better user input feedback.
	- Introduced new interfaces for form values and errors to ensure type safety.
	- Updated forms to include structured layouts with improved accessibility.

- **Bug Fixes**
	- Resolved issues with form submission and error display for various authentication forms.

### CODE REVIEW

**Comment by MuhammadKhalilzadeh** [comment]
@sankettank66 
Would you please add a screen shot of the UI for each page and how it works on these pages?

File: Clients/src/presentation/pages/Authentication/Login/index.tsx [hunk 1]
```diff
@@ -40,110 +62,115 @@ const Login = () => {
...
+            />
+            <Field
+              label="Password"
+              isRequired
+              placeholder="Enter your password"
+              sx={fieldStyles}
+              type="password"
+              value={values.password}
+              onChange={handleChange("password")}
             />
```
[the remark this answers is not in the document]
Comment (sankettank66) [reply Clients/src/presentation/pages/Authentication/Login/index.tsx:108]: As per requirement,
there will be no validation added.

**Review by sankettank66 — COMMENTED** [review]
Hello @MuhammadKhalilzadeh ,
I have reviewed my code. Let me know if there are any changes needed.

**Comment by MuhammadKhalilzadeh** [comment]
@sankettank66  Could you please also add screenshots from your UI for each page?

**Comment by sankettank66** [comment]
## User Interface Screenshots

This comment includes two types of images for various user interface screens: 
1. Screens displaying input values before submission
2. Screens showing logged values in the console after submission

@MuhammadKhalilzadeh Let me know If there are any changes needed.

### Admin Register

- **Before Submission**
  Admin Register - Before

- **After Submission**
  Admin Register - After

---

### Forgot Password

- **Before Submission**
  Forgot Password - Before

- **After Submission**
  Forgot Password - After

---

### Login

- **Before Submission**
  Login - Before

- **After Submission**
  Login - After

---

### Set New Password

- **Before Submission**
  Set New Password - Before

- **After Submission**
  Set New Password - After

---

### User Register

- **Before Submission**
  User Register - Before

- **After Submission**
  User Register - After

---

**Pull request merged**